### PR TITLE
Bump to ISPN 8.0.0

### PR DESCRIPTION
**Pull request by vjuranek**

### CODE REVIEW

**Comment by vjuranek** [comment]
Can we get this merged please? Thanks!

File: wildfly-modules/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM jboss/wildfly:latest
```
Comment (goldmann) [inline wildfly-modules/Dockerfile]: Commenting now on the correct PR :package: 

You should never use `:latest` here. Please specify version here.

**Comment by vjuranek** [comment]
@goldmann PR updated

**Comment by goldmann** [comment]
Build trigerred.

**Pull request merged**